packages/round-manager/src/features/program/ViewProgramPage.tsx

```diff
@@ -109,47 +125,53 @@ export default function ViewProgram() {
 
   return (
     <>
-      <Navbar programCta={true} />
-      <div className="container mx-auto flex flex-col w-screen h-screen">
-        <header className="flex flex-col justify-center border-b pl-2 md:pl-20 py-6">
-          <div className="flex flex-row items-center">
-            <Link to={`/`}>
-              <p className="text-sm text-grey-400 font-semibold">My Programs</p>
-            </Link>
-            <ChevronRightIcon className="h-6 w-6 mx-3 text-sm text-grey-400" aria-hidden="true" />
-            <p className="text-sm text-grey-400 font-semibold">Program Details</p>
-          </div>
-          <h1 className="text-3xl sm:text-[32px] my-2">
-            {program?.metadata?.name || "Program Details"}
-          </h1>
-          {operatorWallets}
-        </header>
-
-        <main className="flex-grow">
-          <div className="px-2 md:px-20 py-3 md:py-6">
-            <div>
-              <div>
-                {isRoundsFetched && roundItems.length > 0 &&
-                  <div className="md:mb-8">
-                    <div className="flex flex-row justify-between">
-                      <p className="font-bold">My Rounds</p>
-                      <Link to={`/round/create?programId=${program?.id}`} className="text-violet-400 font-thin"
-                            data-testid="create-round-small-link">
-                        <PlusSmIcon className="h-5 w-5 inline -translate-y-0.5" aria-hidden="true" /> Create round
-                      </Link>
-                    </div>
-                    {roundItems}
+      {!programExists && <NotFoundPage />}
+      {!hasAccess && <AccessDenied />}
```

Comment:
I noticed that both "Not found" and "No access" pages will both appear on tall enough screens o.0 

i think this is because `hasAccess` is checking `program?.operatorWallets.includes(address?.toLowerCase())`, but this evaluates to a falsy value because of the [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator (`program?.` == `undefined` if `program` is nullish)

![Screen Shot 2022-08-24 at 4 29 34 PM](https://user-images.githubusercontent.com/62616399/186527677-fe36c676-8a9b-418e-9a1a-e7ee99fc04b1.png)

Reply:
Ugh that's ugly. Realized what's happening. Tweaked the condition to ensure only one of those flags are set to false